scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -615,12 +615,11 @@ def summary(self, results, ignore_unrecognized_sections, duration):
                 f'{", " + str(results.none_cases) + " without a status" if results.none_cases else ""}'
                 f' on {len(self.filtered_platforms)} out of total {total_platforms} platforms ({platform_rate:02.2f}%).'
             )
-            if results.skipped_cases or results.filtered_cases or results.notrun_cases:
+            if results.skipped_cases or results.notrun_cases:
                 logger.info(
-                    f'{results.skipped_cases + results.filtered_cases} selected test cases not executed:' \
+                    f'{results.skipped_cases + results.notrun_cases} selected test cases not executed:' \
                     f'{" " + str(results.skipped_cases) + " skipped" if results.skipped_cases else ""}' \
```

Comment:
there is one confusion left in total info with 'skipped' printed here as 'selected':
```
├─ Skipped test cases:   10                   
└─ Selected test cases: 130
...
INFO    - 10 selected test cases not executed: 10 skipped.
```

Reply:
why is there confusion? A selected test can run and be skipped.

Reply:
I guess I know what you mean now, made the change